### PR TITLE
Fix/type import issue

### LINKED ISSUE #233: Issue with @router/common/src... when build SDK
Hello, I'm creating a project with routr/sdk, and when I tried to build always give me this error:

"node_modules/@routr/sdk/dist/client.d.ts:2:44 - error TS2307: Cannot find module '@routr/common/src/connect' or its corresponding type declarations."

2 import { APIClient as ConnectClient } from "@routr/common/src/connect";"

It seems that search about routr/common/src folder and it doesn't exist, like in this line:
https://github.com/fonoster/routr/blob/fc0be57a79d61e96349757e1ddf19cbbe8d6fcff/mods/sdk/src/client.ts#L23

Following the structure about npm package routr/common repo, Should it change dist to src folder maybe? 

Thanks for this awesome SIP server

### PR DESCRIPTION
**Pull request by psanders**
## Description

TL;DR →  Closes #233

This PR resolves a build issue that was observed when working with routr/sdk. The error was pinpointed to an incorrect type import path in routr/mods/sdk/src/client.ts. The faulty path was referencing a src directory, which was not present:

```
node_modules/@routr/sdk/dist/client.d.ts:2:44 - error TS2307: Cannot find module '@routr/common/src/connect' or its corresponding type declarations.
```

After carefully examining the npm package structure of the `@routr/common` repository, it became evident that the correct path should omit the "src" segment. This PR makes that adjustment to ensure successful builds with `@routr/sdk`

> PR comment reviewed with AI

## Type of change

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] This change requires a documentation update

## How Has This Been Tested?

I ran the `npm run build` command and all unit test pass and builds ends without errors.

## Checklist:

- [x] I have performed a self-review of my code
- [ ] I have commented my code, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [x] My changes generate no new warnings
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] New and existing unit tests pass locally with my changes
- [ ] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Comment by psanders** [comment]
@dimusa6 Would you give this a try using the `@routr/sdk@next` version tag? Let me know if it solves the issue for you, so I can merge to main.

**Comment by psanders** [comment]
-- Update -- 

I'm merging the PR since it is passing all test and it feels simple enough.
